spaceship/lib/spaceship/test_flight/build_trains.rb

```diff
@@ -13,12 +13,76 @@ class BuildTrains < Base
     # See `Spaceship::TestFlight::Build#reload`
 
     def self.all(app_id: nil, platform: nil, retry_count: 3)
-      data = client.get_build_trains(app_id: app_id, platform: platform)
-      trains = {}
+      client = Spaceship::ConnectAPI::Base.client
+
+      builds = []
+      included = []
+      cursor = nil
+
+      loop do 
+        builds_resp = client.get_builds(filter: { app: app_id, processingState: "VALID,PROCESSING,FAILED,INVALID"}, limit: 100, sort: "uploadedDate", includes: "buildBetaDetail,betaBuildMetrics,preReleaseVersion,app", cursor: cursor, only_data: false)
+        builds += builds_resp["data"]
+        included += builds_resp["included"]
```

Comment:
If you don't have any builds for a given app id, this ends up crashing because the included field appears to not be included in the response:

```
  /Users/ksmiley/.gem/ruby/2.6.0/bundler/gems/fastlane-f0dbe188ade9/spaceship/lib/spaceship/test_flight/build_trains.rb:25:in `+'
  /Users/ksmiley/.gem/ruby/2.6.0/bundler/gems/fastlane-f0dbe188ade9/spaceship/lib/spaceship/test_flight/build_trains.rb:25:in `block in all'
  /Users/ksmiley/.gem/ruby/2.6.0/bundler/gems/fastlane-f0dbe188ade9/spaceship/lib/spaceship/test_flight/build_trains.rb:22:in `loop'
  /Users/ksmiley/.gem/ruby/2.6.0/bundler/gems/fastlane-f0dbe188ade9/spaceship/lib/spaceship/test_flight/build_trains.rb:22:in `all'
```